### ISSUE TITLE
Java core dumps ibis4oma test omgeving

### ISSUE THREAD

**Issue by sojourn99** [ibissource/iaf#2791]
**❗ please do not add sensitive information in issues, you can provide extra information via email using issue number as reference ❗**

**Describe the issue**
Java core dumps ibis4oma AS test omgeving.
Vergelijkbare Java core dump gezien bij ibs4navigator in productie.
Ibis4oma in test is erg instabiel, veel JVM restarts na core dumps. Deze omgeving wordt veel gebruikt voor dmondemand testen.
Core dumps en verdere info op te vragen bij de indiener van dit issue.

gpf: General Protection Fault in Module=/opt/WAS/java/8.0/jre/lib/amd64/compressedrefs/libj9vm29.so

**Reporter**
Alex Berg

**To Reproduce**
Het is onduidelijk hoe dit probleem gereproduceerd kan worden.

**Screenshots**

**Configuration**
Waarschijnlijk ibis4oma config "realtime", adapter "EngageOneSenderDMOD"
ibis4navigator: adapter "TitanPOST_Subsets_Processor"

**Input**
```
Please provide an example of the input message. 
Alternatively, provide a Ladybug report.
```

**Environment**
FF! 7.7-20220104.143424: ibis4oma 20220106_1012-b179b0d6-ccscon-2522-8
Running on ****** using IBM WebSphere Application Server/9.0.5.7
Java Version: Java(TM) SE Runtime Environment (8.0.6.26 - pxa6480sr6fp26-20210216_01(SR6 FP26))
Heap size: 745M, total JVM memory: 4096M
Free disk space: 10GB, total disk space: 12GB
Up since: 2022-02-16 09:01:37 (5d)

FF! 7.7-20210525.083818: Ibis4Navigator
Running on ********** using IBM WebSphere Application Server/9.0.5.7
Java Version: Java(TM) SE Runtime Environment (8.0.6.26 - pxa6480sr6fp26-20210216_01(SR6 FP26))
Heap size: 2977M, total JVM memory: 8192M
Free disk space: 11GB, total disk space: 12GB
Up since: 2022-02-11 06:44:37 (10d)

**Additional Environment**
 - DBMS: [e.g. Oracle, MSSQL, MariaDB, MySql, PostgreSQL]
 - Browser: [e.g. Chrome, Safari, Edge, Firefox]

**Additional Context**
Add any other context about the problem here. (f.e. ladybug report / test adapter with larva test)

**Comment by jjansenvr** [comment]
Hoi Alex

The problem has its origin in the Eclipse OpenJ9 or IBM J9 which is the JVM you are most probably running on.

I hope you can fix or upgrade the JRE. If you want us to investigate more we can have a look at the logging, (please provide this including all specific Websphere logging to info@wearefrank.nl

**Comment by jacodg** [comment]
@sojourn99 do you have an update? Was the problem reported tot IBM? Did you try upgrading the F!F?

**Comment by sojourn99** [comment]
It seems the issue hasn't come up for quite a while now. We use a FF 7.7 snapshot version, and as I understand the issue is patched in 7.6.1, but could still occur.
I have asked NN WebSphere team to confirm if the issue is reported with IBM.
